armi/reactor/converters/blockConverters.py

```diff
@@ -330,7 +335,12 @@ def __init__(
         self._numInternalRings = numInternalRings
 
     def convert(self):
-        """Return a block converted into cylindrical geometry, possibly with other block types surrounding it."""
+        """Return a block converted into cylindrical geometry, possibly with other block types surrounding it.
+
+        .. impl:: Convert hex blocks to cylindrical blocks
+            :id:  I_ARMI_BLOCKCONV_HEX_TO_CYL1
+            :implements: R_ARMI_BLOCKCONV_HEX_TO_CYL
+        """
```

Comment:
Delete this one.

The one you have lower down in this file on `HexComponentsToCylConverter` is the only one we need.